docker/prod/docker-helpers/install_consul_template.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
If you're using the `metabrainz/python` images now, you can remove all of these scripts under `docker/prod/docker-helpers`.